Hookies/Shared/API/GameplayStore.swift

```diff
@@ -56,8 +56,7 @@ class GameplayStore {
     }
 
     /// Save an instance of gameplay into firestore.
-    func save(gameplay: Gameplay,
-              completion: @escaping (_ gameplay: Gameplay?, _ error: Error?) -> Void) {
+    func save(gameplay: Gameplay) {
```

Comment:
@jinyingtan. I removed completion callback here because the changes made will be made known through the listener.